lib/config/options/index.ts

```diff
@@ -1445,7 +1445,7 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'matchSourceUrls',
-    description: 'A list of source URLs to exact match against.',
+    description: 'A list of source URLs to match against. Each element can be a string, glob pattern or regex',
```

Comment:
I think this file should be charged instead 

https://github.com/renovatebot/renovate/blob/main/docs/usage/configuration-options.md#matchsourceurls

there's already a note about string matching and a link to the additional docs